site.yml

```diff
@@ -34,6 +34,12 @@
   roles:
     - apim
 
+- name: Apply API Management store configuration to an apim node
+  hosts:
+   - apim-is-as-km_1
+  roles:
+   - apim-is-as-km
```

Comment:
Shall we leave the site.yml as it is? If someone needs the IS as Key Manager, they can add it here. Instructions are/should be given in the README.

Reply:
Fixed https://github.com/wso2/ansible-apim/pull/21/commits/25770b373ef66ef006d79d5c6e281b3bd54b0f65